Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM node:lts-alpine AS dependencies
```

Comment:
I guess lts is what we must do to be kind to future self, but let's just be aware of this 🕵️‍♀️

Reply:
Yes. Maybe a little overkill right now, but still the file is quite easy to read and understand I think